subsys/bluetooth/audio/stream.c

```diff
@@ -40,8 +40,10 @@ static uint8_t pack_bt_codec_cc(const struct bt_codec *codec, uint8_t cc[])
 		 * and that based on the Kconfigs we can assume that the length
 		 * will always fit in `cc`
 		 */
-		(void)memcpy(cc + len, data->data, data->data_len);
-		len += data->data_len;
+		cc[len] = data->data_len + 1;
+		cc[len + 1] = data->type;
+		(void)memcpy(cc + len + 2, data->data, data->data_len);
+		len += data->data_len + 2;
```

Comment:
Suggestion to get rid of adding "magic numbers" :) 

```suggestion
		cc[len++] = data->data_len + 1;
		cc[len++] = data->type;
		(void)memcpy(cc + len, data->data, data->data_len);
		len += data->data_len;
```

Reply:
@Thalley Done